tools/gcs-bucket-mover/gcs_bucket_mover/configuration.py

```diff
@@ -0,0 +1,151 @@
+            self._target_project_credentials.service_account_email))
+
+        self._source_storage_client = storage.Client.from_service_account_json(
+            self._source_service_account_json)
+        self._target_storage_client = storage.Client.from_service_account_json(
+            self._target_service_account_json)
+        self._target_service_account_email = self.__get_service_account_email(
+            self._target_storage_client)
+
+        # Load the values passed in on the command line
+        self._bucket_name = conf.bucket_name
+        self._temp_bucket_name = self.__get_temp_bucket_name(
+            self._bucket_name, conf.tempBucketName)
+        self._source_project = conf.source_project
+        self._target_project = conf.target_project
+
+    @property
+    def source_project_credentials(self):
+        """Get the credentials object for the source project"""
+        return self._source_project_credentials
+
+    @property
+    def target_project_credentials(self):
+        """Get the credentials object for the target project"""
+        return self._target_project_credentials
+
+    @property
+    def source_storage_client(self):
+        """Get the storage client object for the source project"""
+        return self._source_storage_client
+
+    @property
+    def target_storage_client(self):
+        """Get the storage client object for the target project"""
+        return self._target_storage_client
+
+    @property
+    def target_service_account_email(self):
+        """Get the email account of the target project's service account"""
+        return self._target_service_account_email
+
+    @property
+    def bucket_name(self):
+        """Get the name of the bucket to be moved"""
+        return self._bucket_name
+
+    @property
+    def temp_bucket_name(self):
+        """Get the temporary bucket name to use in the target project"""
+        return self._temp_bucket_name
+
+    @property
+    def source_project(self):
+        """Get the name of the source project"""
+        return self._source_project
+
+    @property
+    def target_project(self):
+        """Get the name of the target project"""
+        return self._target_project
+
+    def __get_service_account_email(self, storage_client):  #Google guidelines differ from pylint. pylint: disable=no-self-use
+        # Copied from an approved PR to
+        # https://github.com/GoogleCloudPlatform/google-cloud-python/blob/master/storage/google/cloud/storage/client.py pylint: disable=line-too-long
+        # Once it is released, this can be replaced with the library call.
+        """Get the email address of the project's GCS service account
+
+        Args:
+            storage_client: The storage client object used to access GCS
+
+        Returns:
+            The service account email address
+        """
+
+        project = storage_client.project
+        path = '/projects/%s/serviceAccount' % (project,)
```

Comment:
You're mixing use of `.format()` and `%` in the same codebase. Please pick one.
Also this value can be passed directly to the `path` argument below instead of creating a separate variable for a single use